docs/deployment.md

```diff
@@ -0,0 +1,15 @@
+# Push image to DockerHub
+
+1. Log into DockerHub with `docker login`
+2. Build the image with `docker build --file docker/Dockerfile --target production --tag <username>/todo-app:prod --platform linux/amd64 .`
```

Comment:
My app wasn't starting on Azure until I included the platform tag in this step (I'm on a mac)

Reply:
Make sense (as an M1/M2 mac would default to ARM architecture). Good spot!